Tools/Schema.NET.Tool/GeneratorModels/GeneratorSchemaProperty.cs

```diff
@@ -28,7 +28,7 @@ public GeneratorSchemaProperty(GeneratorSchemaClass @class, string jsonName, str
 
     public ICollection<GeneratorSchemaPropertyType> Types { get; } = new List<GeneratorSchemaPropertyType>();
 
-    public IEnumerable<string> CSharpTypes => this.Types.SelectMany(x => x.CSharpTypeStrings);
+    public ICollection<string> CSharpTypes => this.Types.SelectMany(x => x.CSharpTypeStrings).ToList();
```

Comment:
Ditto of my previous comment about `IReadOnlyCollection`/`ToArray`

Reply:
Done